src/lib/expansion/expansion-panel-header.scss

```diff
@@ -0,0 +1,60 @@
+$md-expansion-panel-header-height: 48px;
+$md-expansion-panel-header-height-expanded: 64px;
+
+.mat-expansion-panel-header {
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  height: $md-expansion-panel-header-height;
+  line-height: $md-expansion-panel-header-height;
+  padding: 0 24px;
+
+  .mat-content {
```

Comment:
[Avoid using nested styles](https://github.com/angular/material2/blob/master/CODING_STANDARDS.md#use-lowest-specificity-possible). Should be something like
```scss
.mat-expansion-panel-header {
  // ...
}

.mat-expansion-panel-content {
// ...
}
```